src/main/xsl/preprocess/maprefImpl.xsl

```diff
@@ -389,6 +394,23 @@ See the accompanying LICENSE file for applicable license.
     </xsl:call-template>
   </xsl:template>
 
+  <xsl:template match="@*" mode="preserve-submap-attributes">
+    <xsl:attribute name="dita-ot:submap-{local-name()}" select="."/>
+  </xsl:template>
+  
+  <xsl:template match="*[contains(@class,' topic/title ')]" mode="preserve-submap-title-and-topicmeta">
+    <dita-ot:submap-title class="+ topic/foreign ditaot-d/submap-title ">
```

Comment:
What's the reason for not using `topicmeta/navtitle` instead of `foreign`? It would more closely map to base DITA and would semantically be pretty sound. A problem with that is that if the submap already has a navtitle, then that would be overridden or lost.